### ISSUE TITLE
Vanilla ID Changes

### ISSUE THREAD

**Issue by ErnestoPeroPezzi** [MinecraftForge/MinecraftForge#2316]
Hey, I have been making a modpack for myself and perfectly tuning the whole thing, and I come here with a certain inquiry. I am aware of biome ID conflicts and how to resolve them. However, after installing a certain mod, (Diversity) a biome ID conflict occurred between two vanilla biomes. So this raises the question: 

How would one change Vanilla biome ID's?

**Comment by williewillus** [comment]
Vanilla biome ID's should never change 0.o
What forge version?

**Comment by ErnestoPeroPezzi** [comment]
forge-1.7.10-10.13.4.1566 is what I am using. 

This is the beginning part of the error ;

Conflicting biomes forced game to crash: 

code.elix_x.coremods.antiidconflict.managers.BiomesManager$BiomesIDConflictException: Conflict between biomes with same id caused game to crash.
Affected ids:
140 is asked by: Ice Plains Spikes (net.minecraft.world.biome.BiomeGenSnow), Ice Plains M (biomesoplenty.common.biome.overridden.BiomeGenBOPIcePlains)

Thinking it a bit odd, I used another mod, AntiID conflict to try and to find what the problem was. And this is the response I recieved;

140 is asked by: Ice Plains Spikes (net.minecraft.world.biome.BiomeGenSnow), Ice Plains M (biomesoplenty.common.biome.overridden.BiomeGenBOPIcePlains)

So what exactly can I even do here?
